tests/unit_tests/test_depletion.py

```diff
@@ -59,13 +81,11 @@ def test_get_spent_comps(self):
         First, the materials are defined and then depletion is run to prevent 
         having to store an HDF5 database in the repo
         '''
-        comps = [{922350000:0.04, 922380000:0.96}, 
-                 {551370000:0.1, 360850000:0.8, 541350000:0.1}, 
-                 {942390000:0.10, 942410000:0.9}]
-        self.deplete.run_depletion("examples/", comps)
+        self.deplete.run_depletion("examples/", 10.3)
         spent_comps = self.deplete.get_spent_comps(['5','6','7'], "examples/")
         assert 551370000 in spent_comps[0].keys()
         assert 922350000 in spent_comps[0].keys()
         assert 932410000 not in spent_comps[0].keys()
-        assert spent_comps[0][922350000] == 0.039999352943069735
+        assert spent_comps[0][922350000] == 10.650004036820036
+        assert spent_comps[0][942390000] == 0.22663550016678385
```

Comment:
Might be more robust to have a `reference_depletion_results.h5` that you load in and compare against the spent comps. Not sure if it's that simple with Cyclus.

Reply:
That's probably a cleaner way to do, and more regression style.